config/services.yaml

```diff
@@ -19,7 +19,10 @@ parameters:
     security_advisories_db_repo: 'https://github.com/FriendsOfPHP/security-advisories.git'
     instance_id_file: '%kernel.project_dir%/var/instance-id'
     kernel_version: !php/const Buddy\Repman\Kernel::REPMAN_VERSION
-    storage.aws.endpoint: ~
+    storage.aws.endpoint: ~ # Default to null
+    storage.aws.key: ~
+    storage.aws.secret: ~
+    storage.aws.opaque_auth: false # Default to true
```

Comment:
`false # Default to true` so true or false?

Reply:
Good catch. The default should be `false`, only when explicitly set to `true`, we also want the key and secret.